docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -183,19 +190,15 @@ These errors will not affect the use and functionality of the client, nor will e
 
 ### Webpack 5+: `Uncaught ReferenceError: process is not defined`
 
-Follow the instructions below to if you are using Webpack 5:
-
-<Alert role="none" variation="info">
-*Note:*
-Polyfill is only required for Webpack 5+. In this version Node.js global variable shims required by the `aws-amplify` package were removed, which results in the following error message:
-
-```
-Uncaught ReferenceError: process is not defined
-```
+<Message
+  variation="filled"
+  colorTheme="info"
+  >
+  Note: this issue was fixed in `aws-amplify` version 6. Polyfill is only required for `aws-amplify < 6` and Webpack 5+.
```

Comment:
Little confused abt the second sentence here, is the polyfill only required for projects using `aws-amplify` versions less then 6 **and** `webpack` versions greater than 5 or either?

Reply:
Yeah, hard to word it YES both have to be true. Any suggestions to make it less confusing?

Reply:
Hmmm, yeah kind of head to phrase. What do you think of
```suggestion
  Note: this issue has been fixed in `aws-amplify` version 6. The polyfill is only required for projects that use `aws-amplify` version 5 (or earlier) with Webpack version 5 (or later).
```